ludwig/schema/utils.py

```diff
@@ -65,40 +67,54 @@ def unload_jsonschema_from_marshmallow_class(mclass) -> tDict:
     return schema
 
 
-def InitializerOptions(default: str = "xavier_uniform", description=""):
+# Fields with open signatures (accept any kwargs):
+
+
+def InitializerOptions(default: str = "xavier_uniform", description="", **kwargs):
```

Comment:
Instead of kwargs, could we use `BaseMetadataConfig` directly?

Reply:
yup, this is what I was thinking